pagopa/pn-frontend-e2e
# PN-9223-e-2-e-mittente-sezione-notifiche-paginazione

## Short description

The test didn't have the last step defined in the [confluence page mittente mapping test](https://pagopa.atlassian.net/wiki/spaces/PN/pages/868778282/Mittente+-+Mapping+test-scenari).

## List of changes proposed in this pull request

- added new feature step `And Si clicca sulla pagina numero {int} delle notifiche` in order to change the pagination
- added related method for pagination

## How to test
Simply run mvn clean test -Dcucumber.filter.tags=@TA_MittentePaginazioneNotifiche and verify if the test works fine.
